aireview/src/aireview/prefect/deploy/manage_block.py

```diff
@@ -0,0 +1,112 @@
+import argparse
+import asyncio
+from azure.identity import DefaultAzureCredential
+from azure.keyvault.secrets import SecretClient
+from prefect import get_client, exceptions
+from prefect.blocks.system import Secret
+from prefect_snowflake import SnowflakeCredentials
+
+cli = SecretClient(
+    "https://kvdataprd.vault.azure.net/", credential=DefaultAzureCredential()
+)
+
+
+async def clean_block(block_type: str, block_name: str) -> None:
+    """
+    Remove a block by its type and name
+    e.g. snowflake-credentials:block-snowflake-cred-dpf-dev
+    """
+    cli = get_client()
+    try:
+        id_block = await cli.read_block_document_by_name(block_name, block_type)
+        await cli.delete_block_document(id_block.id)
+    except exceptions.ObjectNotFound:
+        print(f"Already deleted {block_name} block")
+
+
+def deploy_secret(secret_name: str, block_name: str):
+    """Deploy a Key Vault secret to a Prefect Secret block"""
+    sn = secret_name.lower()
```

Comment:
**documentation**: Missing type hints for the function return type. Add '-> None' to maintain consistency with type annotations.